projects/plugins/jetpack/modules/sso.php

```diff
@@ -206,6 +212,68 @@ public function create_error_notice_and_redirect( $query_params ) {
 		return wp_safe_redirect( $url );
 	}
 
+	/**
+	 * Render the invitation email message.
+	 */
+	public function render_invitation_email_message() {
+		// Enqueue the CSS for the admin create user page.
+		wp_enqueue_style( 'jetpack-sso-admin-create-user', plugins_url( 'modules/sso/jetpack-sso-admin-create-user.css', JETPACK__PLUGIN_FILE ), array(), time() );
+
+		wp_admin_notice(
+			__( 'New users will receive an invite to join WordPress.com, so they can log in securely using [Secure Sign On].', 'jetpack' ),
```

Comment:
Secure Sign On should be a link to SSO.

Reply:
Ahh, I misunderstood that, added the link 👍